lib/chai/utils/inspect.js

```diff
@@ -31,15 +31,9 @@ function inspect(obj, showHidden, depth, colors) {
 }
 
 // Returns true if object is a DOM element.
-var isDOMElement = function (object) {
-  if (typeof HTMLElement === 'object') {
+function isDOMElement(object) {
+  if (typeof HTMLElement !== 'undefined') {
```

Comment:
Because of Custom Elements V1, `HTMLElement` is reported as `function` in newest Chrome and Firefox.